src/core/public/chrome/ui/header/recent_items.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React, { useMemo, useState } from 'react';
+import * as Rx from 'rxjs';
+import {
+  EuiPopover,
+  EuiHeaderSectionItemButton,
+  EuiTextColor,
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiTitle,
+  EuiIcon,
+  EuiText,
+} from '@elastic/eui';
+import useObservable from 'react-use/lib/useObservable';
+import { ChromeRecentlyAccessedHistoryItem } from '../..';
+import { WorkspaceObject } from '../../../workspace';
+import { createRecentNavLink } from './nav_link';
+import { HttpStart } from '../../../http';
+import { ChromeNavLink } from '../../../';
+// TODO: replace this icon once added to OUI
+import recent_items from './assets/recent_items.svg';
```

Comment:
Is there a task that tracks this?

Reply:
Yes, i have created one issue to track. https://github.com/opensearch-project/OpenSearch-Dashboards/issues/7354